app/images/moonpay.svg

```diff
@@ -0,0 +1 @@
7A54.4441,54.4441,0,0,0,928.34,191.063a52.2305,52.2305,0,0,0-1.756,13.8888,53.3575,53.3575,0,0,0,1.756,13.8888,51.9043,51.9043,0,0,0,4.7893,11.9731,43.6254,43.6254,0,0,0,7.583,9.8978,42.4692,42.4692,0,0,0,10.217,7.4233,55.6358,55.6358,0,0,0,12.452,4.63,59.3674,59.3674,0,0,0,14.2879,1.6762,60.0757,60.0757,0,0,0,14.4476-1.6762M950.8492,290.2a107.7917,107.7917,0,0,1-23.7068-8.5408,87.3863,87.3863,0,0,1-19.556-13.33,81.6173,81.6173,0,0,1-14.687-17.64,83.8027,83.8027,0,0,1-9.3391-21.392,92.19,92.19,0,0,1,0-48.6906,82.54,82.54,0,0,1,9.3391-21.3121,84.3785,84.3785,0,0,1,14.687-17.64,85.9771,85.9771,0,0,1,19.556-13.41,107.9965,107.9965,0,0,1,23.7068-8.5408,121.5034,121.5034,0,0,1,53.9588,0,104.767,104.767,0,0,1,23.5471,8.5408,91.0054,91.0054,0,0,1,19.556,13.41,84.0413,84.0413,0,0,1,14.9264,17.64,81.05,81.05,0,0,1,9.4189,21.3121,92.1875,92.1875,0,0,1,0,48.6906,82.2766,82.2766,0,0,1-9.4189,21.392,81.3135,81.3135,0,0,1-14.9264,17.64,92.5883,92.5883,0,0,1-19.556,13.33,104.8355,104.8355,0,0,1-23.4673,8.5408,121.8583,121.8583,0,0,1-54.0386,0M773.31,252.7644a57.3171,57.3171,0,0,0,12.5319-4.63,41.9084,41.9084,0,0,0,10.2968-7.4233,47.72,47.72,0,0,0,7.7427-9.8978,43.9413,43.9413,0,0,0,4.7892-11.9731,58.47,58.47,0,0,0,1.5964-13.8888,57.5089,57.5089,0,0,0-1.5964-13.809,45.0831,45.0831,0,0,0-12.5319-21.8708,46.9764,46.9764,0,0,0-10.2968-7.5032,50.4666,50.4666,0,0,0-12.5319-4.7094,62.9775,62.9775,0,0,0-14.4475-1.5964,62.2447,62.2447,0,0,0-14.2879,1.5964,49.0973,49.0973,0,0,0-12.4521,4.7094,47.791,47.791,0,0,0-10.217,7.5032,41.5428,41.5428,0,0,0-7.583,9.8977,54.6382,54.6382,0,0,0-4.7892,11.8933,52.2263,52.2263,0,0,0-1.7561,13.8888,53.3532,53.3532,0,0,0,1.7561,13.8888,52.0786,52.0786,0,0,0,4.7892,11.9731,43.6282,43.6282,0,0,0,7.583,9.8978,42.5124,42.5124,0,0,0,10.217,7.4233,55.6367,55.6367,0,0,0,12.4521,4.63,59.3782,59.3782,0,0,0,14.2879,1.6762,60.0755,60.0755,0,0,0,14.4475-1.6762M732.0427,290.2A107.7909,107.7909,0,0,1,708.336,281.66a87.43,87.43,0,0,1-19.5561-13.33,81.6748,81.6748,0,0,1-14.687-17.64,83.799,83.799,0,0,1-9.339-21.392,92.1875,92.1875,0,0,1,0-48.6906,82.5359,82.5359,0,0,1,9.339-21.3121,84.44,84.44,0,0,1,14.687-17.64,86.02,86.02,0,0,1,19.5561-13.41,107.9957,107.9957,0,0,1,23.7067-8.5408,121.5034,121.5034,0,0,1,53.9588,0,104.73,104.73,0,0,1,23.5471,8.5408,90.9566,90.9566,0,0,1,19.5561,13.41,83.988,83.988,0,0,1,14.9265,17.64,80.9909,80.9909,0,0,1,9.4188,21.3121,92.19,92.19,0,0,1,0,48.6906,82.2161,82.2161,0,0,1-9.4188,21.392,81.2637,81.2637,0,0,1-14.9265,17.64,92.5378,92.5378,0,0,1-19.5561,13.33A104.8355,104.8355,0,0,1,786.0813,290.2a121.8583,121.8583,0,0,1-54.0386,0M372.9434,53.7714h47.8924L502.7318,178.93,584.9472,53.7714H632.84V288.4442H585.4261V127.5257L517.4189,231.2926H488.3641L420.3569,127.5257V288.4442H372.9434Z"/></g><path id="MP_EM_LOGO" class="cls-1" d="M263.5879,95.7848a47.8924,47.8924,0,1,0-47.8924-47.8924,47.8894,47.8894,0,0,0,47.8924,47.8924M116.7378,311.48A116.7378,116.7378,0,1,1,233.4755,194.7426,116.7343,116.7343,0,0,1,116.7378,311.48"/></svg>
```

Comment:
Would be good to add this as a logo component like these [#13953](https://github.com/MetaMask/metamask-extension/pull/13953) so it works with darkmode